core/domain/suggestion_registry_test.py

```diff
@@ -1623,8 +1623,8 @@ def test_validate_change_question_state_data_schema_version(self):
 
         with self.assertRaisesRegexp(
             Exception,
-            'Expected question state schema version to be between 1 and '
-            '%s' % feconf.CURRENT_STATE_SCHEMA_VERSION):
+            'Expected question state schema version to be %s, received %s' % (
```

Comment:
Just put the 0 directly in the error message. No need to interpolate constants.

Reply:
Done!